compose/frontend.yml

```diff
@@ -43,9 +43,9 @@ services:
       - ccd-data-store-api
 
   idam-web-admin:
-    image: hmcts.azurecr.io/hmcts/sidam/idam-web-admin:1.0.0
+    image: "hmctspublic.azurecr.io/idam/web-admin:latest"
     environment:
-      JAVA_OPTS: -Xms128m -Xmx512m
+      JAVA_OPTS: -Xms128m -Xms512m
```

Comment:
same as above

Reply:
yes please revert this change or explain why has been introduced